src/components/20-molecules/file-upload/index.js

```diff
@@ -421,9 +421,8 @@ class AXAFileUpload extends LitElement {
                   multiple
                   @change=${this.handleInputFileChange}
                   variant="red"
-                >
-                  ${this.inputFileText}
-                </axa-input-file>
+                  text="${this.inputFileText}"
```

Comment:
just curious: what motivates this change?

Reply:
Dont know. Just read that in changelog.

Reply:
I know now what motivates it: it's the unreliability of picking up dynamic child text in the old way of specifying input-file text, I myself had advocated to move this to a reliable property.

So, then this change is OK.